### PR TITLE
Targeting for multiple devices

### PR DESCRIPTION
**Pull request by IonutMuthi**

### CODE REVIEW

File: CI/scripts/ports.asv
Comment (SrikanthPagadarai) [inline CI/scripts/ports.asv:1]: File needs to be removed.

File: CI/scripts/synth_designs.sh [hunk 1]
```diff
@@ -5,10 +5,10 @@ MLFLAGS="-nodisplay -nodesktop -nosplash"
 
 if [ -z "$MLRELEASE" ]
 then
-	MLRELEASE=R2022a
+	MLRELEASE=R2021b
 fi
 
-MLPATH=/usr/local/MATLAB
+MLPATH=/emea/mediadata/opt/MATLAB
```
Comment (tfcollins) [inline CI/scripts/synth_designs.sh:11]: Revert since this will break CI

File: hdl/vendor/AnalogDevices/+AnalogDevices/plugin_board.m
Comment (tfcollins) [inline hdl/vendor/AnalogDevices/+AnalogDevices/plugin_board.m:1]: This file removes all the other projects. Please make sure all projects still work

Comment (tfcollins) [inline hdl/vendor/AnalogDevices/+AnalogDevices/plugin_board.m:1]: This file removes all the other projects. Please make sure all projects still work

**Pull request merged**